test/jdk/sanity/client/lib/SwingSet3/src/com/sun/swingset3/demos/table/resources/TableDemo.properties

```diff
@@ -9,14 +9,13 @@ TableDemo.searchLabel=Search Titles and Recipients
 TableDemo.winnersLabel=Show Only Winners
 
 TableDemo.noDataStatusLabel=No data loaded
-TableDemo.loadingStatusLabel=Loading data: 
-TableDemo.rowCountLabel=Showing 
-TableDemo.searchCountLabel=Search found 
+TableDemo.loadingStatusLabel=Loading data:\u0020
```

Comment:
According to https://github.com/openjdk/jdk/pull/11488/files#r1038605801 the latter two are actually needed as spaces, and the first might be; so keeping it as well seems to be the safe choice.